docs/standard/memory-and-spans/memory-t-usage-guidelines.md

```diff
@@ -235,7 +235,7 @@ Accepting an instance of this type signals that your component intends to take o
 Any component that transfers ownership of the <xref:System.Buffers.IMemoryOwner%601> instance to a different component should no longer use that instance after the method call completes.
 
 > [!IMPORTANT]
-> If your constructor accepts <xref:System.Buffers.IMemoryOwner%601> as a parameter, its type should implement <xref:System.IDisposable>, and your <xref:System.IDisposable.Dispose%2A> method should call <xref:System.Buffers.MemoryPool%601.Dispose%2A?displayProperty=nameWithType>.
+> If your constructor accepts <xref:System.Buffers.IMemoryOwner%601> as a parameter, its type should implement <xref:System.IDisposable>, and your <xref:System.IDisposable.Dispose%2A> method should call <xref:System.Buffers.IMemoryOwner%601.Dispose%2A?displayProperty=nameWithType>.
```

Comment:
That link doesn't work because Dispose is a member of IMemoryOwner only by inheritance. MemoryPool might be correct here. I'll defer to @GrabYourPitchforks

Reply:
How about writing `IMemoryOwner.Dispose()` without linking? MemoryPool is not an `IMemoryOwner` AFAIK, and `IMemoryOwner` can be implemented by other classes.

Reply:
Or 

> ... call `Dispose` on the `IMemoryOwner` object.

?

Reply:
Fixed.